examples/constant-force-optical-trap/extract-data.py

```diff
@@ -40,56 +40,56 @@
 
 # process all datasets
 for dataset in datasets:
-    print("Extracting data from dataset '%s'..." % dataset)
+    print("Extracting data from dataset '{}'...".format(dataset))
 
     # Open Excel spreadsheet file.
     workbook_filename = os.path.join(original_data_directory, dataset + '_data.xls')
     workbook = xlrd.open_workbook(workbook_filename)
 
     # DEBUG
-    print("Workbook contains %d worksheets:" % workbook.nsheets)
+    print("Workbook contains {:d} worksheets:",format(workbook.nsheets))
```

Comment:
This should be `.format(...)` (notice the period instead of the comma)

Reply:
Thanks, will fix, I think that this code didn't execute because of missing a dependency, and I didn't worry, since the example runs without it (data already extracted).